static/views/edit-developer-profile.html

```diff
@@ -14,326 +14,416 @@
+        {{ if .LoggedUser }}<a style="text-decoration: underline;" href="/profile/home">Your Profile</a>{{ else }}<a href="/auth">Login</a>{{ end }}
+        </div>
+        <div style="clear: both;"></div>
+      </nav>
+      </header>
+    <div id="spinner-0">
+      <div class="overlay-effect"></div>
+      <div class="spinner-box"><div class="lds-ring"><div></div><div></div><div></div><div></div></div></div>
+    </div>
+    <section>
+      <article>
+        <p>
+        <h3>Your Profile is Live on {{ .SiteName }}</h3>
+        <small>
+          <b>Created:</b> {{ .DeveloperProfile.CreatedAt.Format "Jan 02, 2006 15:04:05 UTC" }}<br />
+          <b>Last Updated:</b> {{ .DeveloperProfile.UpdatedAt.Format "Jan 02, 2006 15:04:05 UTC" }}<br />
+          <b>View Profile:</b> <a href="/developer/{{ .DeveloperProfile.Slug }}" rel="noopener noreferrer" target="_blank">https://{{ .SiteHost }}/developer/{{ .DeveloperProfile.Slug }}</a>
+        </small><br /><br />
+        </p>
+      </article>
+      <article style="margin-top: 30px;">
+        <p>
+        <h3>Edit Your Developer Profile</h3>
+        <input type="hidden" name="profile-id" id="profile-id" value="{{ .DeveloperProfile.ID }}"/><br />
+        <input type="text" name="full-name" id="full-name" placeholder="Full Name" style="width: 100%;" value="{{ .DeveloperProfile.Name }}"/><br />
+        <input type="text" name="location" id="location" placeholder="Job Location" style="width: 100%;" value="{{ .DeveloperProfile.Location }}"/><br />
+        <input type="number" name="hourly-rate" id="hourly-rate" placeholder="Hourly Rate in USD" style="width: 10%;" value="{{ if ne .DeveloperProfile.HourlyRate 0 }}{{ .DeveloperProfile.HourlyRate }}{{ end }}"/> USD/hour<br />
+        <input type="url" name="linkedin-url" id="linkedin-url" placeholder="Linkedin URL" style="width: 100%;" value="{{ .DeveloperProfile.LinkedinURL }}"/><br />
+        <div>
+          <label for="profile-image" style="float: left;cursor: pointer;background: {{ .PrimaryColor }};color: #fff;padding: 6.525px 23.4px;border-radius: 3.6px;">Upload Profile Image</label>
+          <div style="margin-bottom: 10px;float:left; width:100%;">
+            <img id="profile-image-preview" src="/x/s/m/{{ .DeveloperProfile.ImageID }}" style="max-width:50%;cursor: pointer;float: left;"/>
+          </div>
+          <input type="file" name="profile-image" id="profile-image" placeholder="Profile Image" style="visibility: hidden; width: 0; height: 0;" />
+          <div class="clearfix"></div>
+        </div>
+        <br />
+        <input type="hidden" name="profile-image-id" id="profile-image-id" value="{{ .DeveloperProfile.ImageID }}" />
+        <div id="search-tag-container">
+              <label for="search-tag">Skills</label>
+              <input autocomplete="off" type="text" placeholder="Skills" id="search-tag"/>
+              <button onclick="addSkillTag()" disabled id="add-tag-btn">Add</button>
```

Comment:
<img width="1144" alt="image" src="https://user-images.githubusercontent.com/70442266/231588754-034f4ce5-5680-461f-98da-ad1f5756778c.png">

- the autocomplete comes out of the page it should be the same width as the input box
- can you please change the color of the 'Add' button to be same as 'Save' and all primary colors?
- I originally asked for the skills to be upper case but i think it will be better to keep them with their original casing to avoid confusion. I will be updating the database with an updated list

Reply:
1. Autocomplete css fixed.
2. Add button css fixed.
3. The skills shown here will be in same casing as saved in DB, I am not doing an upper in code at all.
3. Added skills to submit developer page as well.

Reply:
Submit developer profile
<img width="946" alt="image" src="https://user-images.githubusercontent.com/129782720/232507325-8da7f7da-5ab1-4af7-bf36-cd0f4b4e035d.png">